tornado-runtime/src/main/java/uk/ac/manchester/tornado/runtime/tasks/meta/TaskMetaData.java

```diff
@@ -424,8 +424,8 @@ public boolean isCpuConfigDefined() {
         return super.isCpuConfigDefined() || scheduleMetaData.isCpuConfigDefined();
     }
 
-    @Override
-    public String toString() {
-        return String.format("task meta data: domain=%s, global workgroup size=%s%n", domain, (getGlobalWork() == null) ? "null" : formatWorkDimensionArray(getGlobalWork(), "1"));
-    }
+    //    @Override
```

Comment:
Why is this commented out?

Reply:
Good catch. This was for debugging. I will revert this.